app/components/articleSearch/components/searchItem/infoList.tsx

```diff
@@ -0,0 +1,85 @@
+import * as React from "react";
+import { trackAndOpenLink } from "../../../../helpers/handleGA";
+import Icon from "../../../../icons";
+import alertToast from "../../../../helpers/makePlutoToastAction";
+
+const styles = require("./infoList.scss");
+
+export interface IInfoListProps {
+  referenceCount: number;
+  citedCount: number;
+  citedPaperAvgIF: number;
+  plutoScore: number;
+  source: string;
+  DOI: string;
+  articleId: number;
+}
+
+function copyDOI(DOI: string) {
+  const textField = document.createElement("textarea");
```

Comment:
:+1:

Reply:
I wonder it works on every browser.

Reply:
https://caniuse.com/#search=document.execCommand()

The document.execCommand() method is not supported in IE9 and earlier.
So I judged that it can be used.